programs/survival/graves.sc

```diff
@@ -0,0 +1,349 @@
+// Tweak Stuff
```

Comment:
You can add your name as well as a small description up at the top

Reply:
Aye will do